app/js/pages/Berlin2018Page.js

```diff
@@ -0,0 +1,184 @@
+'use strict'
+
+import React, { Component, Fragment } from 'react'
+import { Helmet } from 'react-helmet'
+import CollapsibleList from '../components/CollapsibleList'
+import berlinData from '../datastore/berlin-event-data'
+import Slider from 'react-slick'
+import SignatureTalks from '../components/SignatureTalks'
+import { slugify } from '../utils/functions'
+
+const settings = {
+  className: 'photo-slider center',
+  centerMode: true,
+  infinite: true,
+  centerPadding: '0',
+  slidesToShow: 3,
+  slidesToScroll: 1,
+  autoPlay: true,
+  autoplaySpeed: 1500,
+  initialSlide: 1,
+  speed: 500,
+  arrows: true,
+  dots: true,
+  useCSS: true,
+  accessibility: true,
+  responsive: [
+    {
+      breakpoint: 1024,
+      settings: {
+        slidesToShow: 2,
+      },
+    },
+    {
+      breakpoint: 600,
+      settings: {
+        slidesToShow: 1,
+      },
+    },
+  ],
+}
+
+const photoSlides = [
+  {
+    src:
+      'https://blockstack.imgix.net/ce1ee40f-d333-4778-ac7d-62d1231a3a9c__3020095.jpg',
+  },
+
+  {
+    src:
+      'https://blockstack.imgix.net/9f001bca-23bb-4510-9f0b-4e166514f60c__3020273.jpg',
+  },
+  {
+    src:
+      'https://blockstack.imgix.net/6fa2356d-07fd-4ee7-a083-4d8d80dce521_p3010031.jpg',
+  },
+  {
+    src:
+      'https://blockstack.imgix.net/b0415647-dc2f-44ac-b083-e7f445229f10__3020103.jpg',
+  },
+  {
+    src:
+      'https://blockstack.imgix.net/2a992e79-07b5-4440-a681-6c33a339a5ad__3020104.jpg',
+  },
+]
+
+function removeDuplicates(myArr, prop) {
+  return myArr.filter((obj, pos, arr) => {
+    return arr.map((mapObj) => mapObj[prop]).indexOf(obj[prop]) === pos
+  })
+}
+
+const filterIndex = (data, min, max) =>
+  data.filter((item, i) => {
+    if (min && max) {
+      return i >= min && i <= max
+    }
+    if (min && !max) {
+      return i >= min
+    }
+    if (!min && max) {
+      return i <= max
+    }
+  })
+
+const filteredData = [...removeDuplicates(berlinData, 'talkTitle')]
+
+const mainEvent = filteredData.filter((day) => day.day === 'March 2')
+
+const sections = [
+  {
+    title: 'Morning Sessions',
+    items: filterIndex(mainEvent, 0, 8),
+  },
+  {
+    title: 'After Lunch Sessions',
+    items: filterIndex(mainEvent, 9, 11),
+  },
+  {
+    title: 'App Demos',
+    items: filterIndex(mainEvent, 12, 23),
+  },
+  {
+    title: 'Late Afternoon Sessions',
+    items: filterIndex(mainEvent, 24, 26),
```

Comment:
I like how thoughtful this is laid out codewise. The `filterIndex` idea feels a bit brittle to me. Can we quickly switch this to filtering on a new key that we add to events? For instance:
```
items: mainEvent.filter(item => item.type === 'lateAfternoonSession')
```